datasets/ho/make_data.py

```diff
@@ -81,25 +85,27 @@
 points3d_test = []
 
 # Evaluation
-for subject in sorted(evaluation_root_path.iterdir()):
-    rgb = Path(subject, 'rgb')
-    meta = Path(subject, 'meta')
-    for rgb_file in sorted(rgb.iterdir()):
-        meta_file = Path(meta, f'{rgb_file.stem}.pkl')
-        data = np.load(meta_file, allow_pickle=True)
-
-        cam_intr = data['camMat']
-        hand3d = np.repeat(np.expand_dims(data['handJoints3D'], 0), 21, 0)
-        hand3d = hand3d[reorder_idx]
-        obj_corners = data['objCorners3D']
-        hand_object3d = np.concatenate([hand3d, obj_corners])
-        hand_object3d = hand_object3d.dot(coordChangeMat.T)
-        hand_object_proj = cam_intr.dot(hand_object3d.transpose()).transpose()
-        hand_object2d = (hand_object_proj / hand_object_proj[:, 2:])[:, :2]
 
-        images_test.append(rgb_file)
-        points3d_test.append(hand_object3d)
-        points2d_test.append(hand_object2d)
```

Comment:
Why was the `for` loop removed here?  It still references `subject` so it likely is just taking the last value that `subject` was from ... somewhere